pyproject.toml

```
@@ -38,6 +38,7 @@ dependencies = [
     "azure-identity>=1.12.0",
     "azure-ai-ml==1.13.0",
     "azure-storage-blob>=12.19.0",
+    "confusables>=1.2.0",
```

With smaller projects, we've been pinning the version so we can do some basic safety checks (e.g. supply chain attack)

```suggestion
    "confusables==1.2.0",
```

---

Good call, done.